src/containers/LectureDetailContainers/LectureClassContainer.js

```diff
@@ -0,0 +1,230 @@
+import TimetableModal from "components/LectureDetailComponents/TimetableModal";
+import {
+  openTimetableModal,
+  closeTimetableModal,
+  setLectureTimetables,
+} from "store/modules/lectureDetailModule";
+import { getValueOnLocalStorage } from "utils/localStorageUtils";
+import { triggerWhenNotLoggedIn } from "utils/reportUtils";
+import { classTime } from "static/LecturesDetailPage/classTime";
+
+const Section = styled.section`
+  width: 368px;
+  height: fit-content;
+  min-height: 381px;
+  grid-column: 8 / 12;
+  padding: 24px;
+  border-radius: 8px;
+  border: solid 1px ${BorderColor};
+`;
+
+const Wrapper = styled.section`
+  min-height: 320px;
+`;
+
+const InfoLabel = styled.div`
+  display: block;
+
+  margin: 0 10px 24px 0;
+  color: ${FontColor};
+  font-size: 20px;
+  font-weight: 500;
+`;
+
+const SubInfoLabel = styled.p`
+  margin-bottom: 16px;
+`;
+
+const SubWarningWrapper = styled.div`
+  display: flex;
+  height: 327px;
+  justify-content: center;
+  align-items: center;
+`;
+const SubWarningLabel = styled.p`
+  text-align: center;
+  font-size: 12px;
+  line-height: normal;
+  letter-spacing: normal;
+  color: ${PlaceholderColor};
+`;
+
+const SubLabel = styled.label`
+  margin: 16px 16px 17px 0;
+  font-size: 14px;
+  font-weight: normal;
+  font-style: normal;
+  line-height: normal;
+  color: ${PlaceholderColor};
+`;
+const SubLabelContent = styled.label`
+  font-size: 14px;
+  font-weight: normal;
+  font-style: normal;
+  line-height: normal;
+  color: ${FontColor};
+`;
+const ClassContent = styled.div`
+  margin-bottom: 13px;
+  height: 28px;
+  line-height: 28px;
+  margin-left: 40px;
+`;
+const AddTimtable = styled.button`
+  float: right;
+  width: 56px;
+  height: 28px;
+
+  padding: 5px 16px 5px 17px;
+
+  border: none;
+  color: #fff;
+  border-radius: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  cursor: pointer;
+  background-color: ${ConceptColor};
+  :before {
+    content: "담기";
+  }
+`;
+const RemoveTimetable = styled(AddTimtable)`
+  background-color: #ffab2e;
+  :before {
+    content: "빼기";
+  }
+`;
+
+/**
+ * 인자로 받은 배열의 첫번째 요소로 요일을 가져오는 함수입니다.
+ * 배열의 첫 번째로 들어온 수의 가장 앞자라리로 classTime 배열안에서 해당 요일이 위치해 있는 배열의 수를 반환합니다.
+ * classTime.js
+ * @param {*} classTimes 수업 시간 배열
+ * @returns 수업 시작~끝 문자열로 조합한 결과
+ */
+const getDay = (classTimes) => {
+  let dayNo = 2 + parseInt(parseInt(classTimes.split(",")[0].replace(/\[/, "")) / 100);
+  return classTime[0][dayNo] + " ";
+};
+
+/**
+ * 인자로 받은 배열의 첫번째와 마지막 요소로 수업 시간을 문자열로 조합해 반환하는 함수입니다.
+ * 인자가 문자열로 들어와서 배열로 바꿔준 뒤 첫번째와 마지막 요소로 classTime에서 수업 시간을 가져와
+ * 문자열로 조합한 결과를 반환합니다.
+ * @param {*} classTimes 수업 시간 배열
+ * @returns 수업 시작~끝 문자열로 조합한 결과
+ */
+const classStartToEnd = (classTimes) => {
+  let times = classTimes.replace(/\[/, "").replace(/\]/, "").split(","),
+    start = 1 + parseInt(times[0] % 100),
+    end = 1 + parseInt(times[times.length - 1] % 100);
+
+  return classTime[start][0] + "~" + classTime[end][0];
+};
```

Comment:
함수를 보니 어떤 역할을 하는 함수인지는 알겠습니다만... 
다영님은 classStartToEnd를 봤을 때 무슨 함수인지 예측할 수 있으신가요

```suggestion
const getClassTimePeriod = (classTimes) => {
  let times = classTimes.replace(/\[/, "").replace(/\]/, "").split(","),
    start = 1 + parseInt(times[0] % 100),
    end = 1 + parseInt(times[times.length - 1] % 100);

  return classTime[start][0] + "~" + classTime[end][0];
};
```